report_thesis/src/sections/experiment_design/initial_experiment.tex

```diff
@@ -0,0 +1,18 @@
+\subsection{Design for Initial Experiment}\label{sec:initial-experiment}
+The goal of the initial experiment is to identify the most promising models for further evaluation.
+This experiment was designed to give an initial indication of the performance of the models on the \gls{ccs} dataset.
+
+To conduct the initial experiment and evaluate the twelve selected models shown in Table~\ref{tab:preprocessing-models} we wanted to ensure that the models were evaluated on a consistent basis.
+We achieve this by setting the following conditions for the experiment.
+The only preprocessor used for all models was \texttt{Norm 3} normalization.
+This normalization technique was developed specifically by \citet{cleggRecalibrationMarsScience2017} for accomodating the characteristics of the \gls{libs} data, whereby the data represents the readings of three different spectrometers, as outlined in Section~\ref{sec:norm3}.
+Therefore, \texttt{Norm 3} represents the most \gls{libs}-specific normalization technique available in from the pool of selected preprocessors and is the most appropriate choice for this experiment.
```

Comment:
Past tense + it isn't a characteristic of LIBS data, but the collection process. 
They have 5 spectrometers on SuperCam, and use Norm 5 there. It's a general approach.

```suggestion
This normalization technique was developed by \citet{cleggRecalibrationMarsScience2017} to accommodate the characteristics of how the \gls{libs} data is collected, whereby the data represented the readings of three different spectrometers, as outlined in Section~\ref{sec:norm3}.
Therefore, \texttt{Norm 3} represented the most \gls{libs}-specific normalization technique available from the pool of selected preprocessors and was the most appropriate choice for this experiment.
```

Reply:
> Therefore, \texttt{Norm 3} represented the most \gls{libs}-specific normalization technique available from the pool of selected preprocessors and was the most appropriate choice for this experiment.

I'm not sure how I feel about this argument. Not great, but I don't really have anything much better. Can we discuss?

Reply:
Same here, the intro was changed.